libkbfs/folder_block_ops.go

```diff
@@ -2585,14 +2575,36 @@ func (fbo *folderBlockOps) getDeferredWriteCountForTest(lState *lockState) int {
 	return len(fbo.deferredWrites)
 }
 
+func (fbo *folderBlockOps) updatePointer(kmd KeyMetadata, oldPtr BlockPointer, newPtr BlockPointer) {
+	updated := fbo.nodeCache.UpdatePointer(oldPtr.Ref(), newPtr)
+	if !updated {
+		return
+	}
+	// Prefetch the new reference, but only if it already exists in the block
```

Comment:
But that's fine, right? The idea here is to respect the LRU of the block cache with respect to the old reference. The only reason I can think of to skip this check is if a pointer update increases the probability that a client will demand a block in the near future. WDYT?